### DISCUSSION TITLE
Genesenennachweis - Falsches Datum

### DISCUSSION THREAD

**Discussion by ruiyan-yu** [Digitaler-Impfnachweis/certification-apis#108] [Q&A] [answered]
Hallo,

Das Feld "Datum des ersten positiven Testergebnisses" auf der PDF-Datei ist falsch.

![FELD](https://user-images.githubusercontent.com/18198895/122240456-166ab000-cec2-11eb-8107-1eb98aa4eb6b.PNG)

Mit freundlichen Grüßen

Duria e.G

**Comment by thinkberg** [comment]
Wir geprueft.

**Comment by thinkberg** [comment, marked as answer]
Wird gefixt und steht bald auf der Test-stage zur Verfuegung.

**Comment by oliver-steinbrecher** [comment]
@thinkberg has this been fixed?

**Reply by thinkberg** [reply]
This was only on the test stage and has been fixed.

**Reply by dirkexe** [reply]
The bug still exists in the production environment. Just tested:
<img width="396" alt="Bildschirmfoto 2021-06-23 um 08 04 21" src="https://user-images.githubusercontent.com/8550081/123044607-2de1f580-d3fa-11eb-9e38-fda62d566f45.png">

**Reply by thinkberg** [reply]
Wir haben eben einen Hotfix eingespielt, da ein anderes Template auch betroffen war und haben dieses Template auch korrigiert.

**Reply by D-Winkler** [reply]
Von einem Kunden bestätigt, der Fehler ist raus.
